internal/wire/parse.go

```diff
@@ -753,7 +753,7 @@ func processStructLiteralProvider(fset *token.FileSet, typeName *types.TypeName)
 
 	pos := typeName.Pos()
 	fmt.Fprintf(os.Stderr,
-		"Deprecated: %v, see https://godoc.org/github.com/google/wire#Struct for more information.",
+		"Deprecated: %v, see https://godoc.org/github.com/google/wire#Struct for more information.\n",
```

Comment:
I suggest either:

`Warning: %v is deprecated and will be removed in the next release; see https://godoc.org/github.com/google/wire#Struct for more information.\n`

or just changing `Deprecated` to `Deprecation warning: `.